src/index.js

```diff
@@ -25,14 +25,180 @@ if (typeof AudioWorkletNode !== 'function') {
     const outputChannels = options && options.outputChannelCount ? options.outputChannelCount[0] : 2;
     const scriptProcessor = context.createScriptProcessor(undefined, 2, outputChannels);
 
+    const automationEvents = new Map();
+
+    function SetValue (value, startTime) {
```

Comment:
Seems like these classes could get pulled out of `AudioWorkletNode` so they're shared?